python/src/opendp/mod.py

```diff
@@ -425,12 +426,6 @@ def eval(self, query):
         from opendp.core import queryable_eval # pragma: no cover
         return queryable_eval(self.value, query) # pragma: no cover
 
-    @property
-    def query_type(self):
-        from opendp.core import queryable_query_type
-        from opendp.typing import RuntimeType
-        return RuntimeType.parse(queryable_query_type(self.value))
```

Comment:
Oh! It looks like you had a wrapper around `RuntimeType.parse` before, and are moving away from that. There's probably a good reason, but I don't understand the motivation.

Reply:
The query type of a queryable will never change (it is static), so there's no need to repeatedly retrieve the query type at every call. Since I already need to know the query type to convey it into the correct Python type (Queryable or OnceFrame), I might as well simply the property to just an attribute.